doozerlib/runtime.py

```diff
@@ -398,7 +398,19 @@ def filter_disabled(n, d):
             exclude_keys = flatten_list(self.exclude)
             image_ex = list(exclude_keys)
             rpm_ex = list(exclude_keys)
-            image_keys = flatten_list(self.images)
+            image_keys = []
+            self.pinned_commitishes = {}
+            # `image` can either be a `distgit key` or `distgit_key:commit-ish`
+            for image in flatten_list(self.images):
+                image_split = image.split(":")
+                distgit_key = image_split[0]
+                image_keys.append(distgit_key)
+                if len(image_split) < 2:
+                    continue
+                commitish = image_split[1]
+                # TODO: verify `commitish` format
```

Comment:
meh... I think it's fine not to.

Reply:
Well, now I think we'd better not to implement that check on Doozer. Doozer should be able to build from anywhere. We can implement the check in the component invoking Doozer, like a Jenkins job.